lib/sequences/conformance_sequence.rb

```diff
@@ -11,8 +11,10 @@ class ConformanceSequence < SequenceBase
     assert @conformance.class == FHIR::DSTU2::Conformance, 'Expected valid DSTU2 Conformance resource'
   end
 
-  test 'Conformance states proper json support' do
-    assert @conformance.format.include?('application/json+fhir'), 'Conformance statement does not state support for applciation/json+fhir.'
+  test 'Conformance states proper JSON or XML support' do
```

Comment:
Under "Description" at the Argonaut Data Query Implementation Guide Server, there are notes stating that JSON support is required by XML support is preferred but optional. The test was listed for only JSON support because of that. I think we should probably break this down into two tests, with the JSON test marking failure if unsupported and the XML test marking a warning if unsupported.

Link to documentation: http://www.fhir.org/guides/argonaut/r2/Conformance-server.html